CMakeLists.txt

```
@@ -22,8 +22,9 @@ FIND_PACKAGE( ILCUTIL REQUIRED COMPONENTS ILCSOFT_CMAKE_MODULES )
 INCLUDE( ilcsoft_default_settings )
 
 FIND_PACKAGE( Marlin 1.0 REQUIRED ) # minimum required Marlin version
+FIND_PACKAGE( LCIO REQUIRED )
 INCLUDE_DIRECTORIES( SYSTEM ${Marlin_INCLUDE_DIRS} )
-LINK_LIBRARIES( ${Marlin_LIBRARIES} )
+LINK_LIBRARIES( LCIO::lcio Marlin streamlog )
```

```suggestion
LINK_LIBRARIES( LCIO::lcio ${Marlin_LIBRARIES} )
```
Marlin does not yet have targets, I think, so we have to go the old school way here.